flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/TableEnvHiveConnectorITCase.java

```diff
@@ -531,6 +532,52 @@ public void testLocationWithComma() throws Exception {
         }
     }
 
+    @Test
+    public void testReadHiveDataWithEmptyMapForHiveShim20X() throws Exception {
+        TableEnvironment tableEnv = getTableEnvWithHiveCatalog();
+
+        try {
+            // Flink to write parquet file
```

Comment:
Does this issue only happen with Parquet tables? If not, we can just use text table so that the test can be simpler.

Reply:
yes，parquet